manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "AI-on-the-edge",
+  "version": "12.0.1",
+  "funding_url": "https://www.paypal.com/donate?hosted_button_id=8TRSVYNYKDSWL",
+  "new_install_prompt_erase": false,
+  "builds": [
+    {
+      "chipFamily": "ESP32",
+      "parts": [
+        {
+          "path": "releases/download/v11.2.0/bootloader.bin",
```

Comment:
Instead of having to add the binaries to the repo, can we refer to an URL, eg. https://github.com/jomjol/AI-on-the-edge-device/releases/download/v12.0.1/firmware.bin?
If this would work it would make maintenance easier. The needed binaries can be added to the release if needed.

Reply:
I had a hard time with referencing them directly from my fork. (Cross origin stuff)

You could an alpha folder were you could experiment with the direct links in the manifest though.